script/chests.js

```diff
@@ -48,14 +48,26 @@ function lens(loc){
     }
 }
 
+function hasboom(){
+    if(chuInLogic == true){
+    //    alert('chus in logic')
```

Comment:
Any particular reason these comments are here?

Reply:
forgot to stage that when removing TODO and commented code. Just a debug alert, IMO easier to use than debug. Will Fix..